modules/ipi-mirroring-a-disconnected-registry.adoc

```diff
@@ -0,0 +1,244 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// install/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-mirroring-a-disconnected-registry_{context}"]
+
+= Mirroring the repository (optional)
+
+.Prerequisites
+
+* You have installed OpenShift CLI (`oc`).
+* You have a designated mirror registry.
+* (Optional) You have installed the `oc-mirror-plugin` tool.
+
+.Procedure
+
+Complete the following steps on the mirror host:
+
+. Review the
+link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
+to determine the version of {product-title} that you want to install and determine the corresponding tag on the link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Repository Tags] page.
+
+. Set the required environment variables:
+.. Export the release version:
++
+[source,terminal]
+----
+$ OCP_RELEASE=<release_version>
```

Comment:
I'm somewhat concerned about the vast amount of duplication between this topic and https://github.com/openshift/openshift-docs/blob/main/modules/installation-mirror-repository.adoc.

Is this different enough to _not_ be able to conditionalize the differences?